src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -281,15 +309,62 @@ private void EnsureGlobbingUrlBuilder()
                    .Append("</script>");
         }
 
-        private void AppendSrc(TagHelperContent content, string srcKey, string srcValue)
+        private void AppendSrc(TagHelperContent content, string srcKey, string srcValue, bool escapeQuotes)
```

Comment:
what does not escaping quotes for the `src` attribute value have to do with adding the file version?  fine to do a couple of small unrelated changes while you have the file open but this PR seems to mix at least three user-visible changes.